AmericanExpress/src/main/java/com/braintreepayments/api/americanexpress/AmericanExpressRewardsBalance.kt

```diff
@@ -0,0 +1,144 @@
+package com.braintreepayments.api.americanexpress
+
+import android.os.Parcel
+import android.os.Parcelable
+import androidx.annotation.RestrictTo
+import com.braintreepayments.api.sharedutils.Json
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Class to parse and contain American Express rewards balance
+ */
+class AmericanExpressRewardsBalance : Parcelable {
```

Comment:
Can we use the @Parcelize annotation here instead? I think that should remove the need to define the parcelable implementation in this class

Reply:
+1